src/createReducer.ts

```diff
@@ -7,6 +7,11 @@ import {
 
 /**
  * Defines a mapping from action types to corresponding action object shapes.
+ *
+ * @deprecated This should not be used manually - it is only used for internal
+ *             interference purposes and should not have any further value.
```

Comment:
Should this be `inference` instead of `interference`?

Reply:
Yup. Thanks for catching!